api/server/router/image/image_routes.go

```diff
@@ -40,8 +41,10 @@ func (s *imageRouter) postImagesCreate(ctx context.Context, w http.ResponseWrite
 	)
 	defer output.Close()
 
-	w.Header().Set("Content-Type", httputil.NegotiateContentType(r,
-		[]string{"application/json-seq", "application/json"}, "application/json"))
+	mediaType := httputil.NegotiateContentType(r,
+		[]string{api.MediaTypeJsonSequence, "application/json"}, "application/json")
+	w.Header().Set("Content-Type", mediaType)
+	out := types.WithMediaType(output, mediaType)
```

Comment:
Perhaps it makes sense to define a function or variable for this in the streamformatter (`streamformatter.MediaTypes`), as they'll now always be used together. (Perhaps we should even have a function that includes the negotiation, although not sure if that makes most sense in streamformater, or in the API 🤔)